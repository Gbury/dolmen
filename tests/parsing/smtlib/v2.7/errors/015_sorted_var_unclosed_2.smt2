(set-logic ALL)
(assert (forall ((x int foo)) x))
