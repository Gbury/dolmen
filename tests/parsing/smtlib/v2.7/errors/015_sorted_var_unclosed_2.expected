File "tests/parsing/smtlib/v2.7/errors/015_sorted_var_unclosed_2.smt2", line 2, character 24-27:
2 | (assert (forall ((x int foo)) x))
                            ^^^
Error while parsing a sorted variable, read the symbol 'foo',
      but expected a closing parenthesis.
